src/ValidatorsList.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import Store from './store'
+
+import ValidatorsActions from './actions/validators'
+
+class ValidatorsList extends Component {
+  componentWillMount() {
+    Store.dispatch(ValidatorsActions.findAll())
+  }
+
+  render() {
+    const validators = this.props.validators.list
+    return (
+      <div>
+        <p>Validators:</p>
+        { validators.length === 0 ? <em>No validators</em> : <ul>{this._buildValidatorsList(validators)}</ul>}
```

Comment:
I'd prefer to have this in separate lines.

Reply:
Please take a look at the commit I pushed, and let  me know if that's what you meant.